### PR TITLE
Created a README for DocumentViewer

### PR DESCRIPTION
**Pull request by Nova281**
created a README for DocumentViewer. includes a short description and screenshot with pdf on it.

### CODE REVIEW

**Comment by Nova281** [comment]
Update README.md is the latest update to the README.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+![DocumentViewer icon](http://i68.tinypic.com/2rpxv87.png)DocumentViewer
+=
+
+DocumentViewer is a viewer supporting PDF and DJVU files. 
+Many features are present, making it easier to open and view documents.
```
Comment (humdingerb) [inline README.md]: Better: "It provides many features , making it easy to open and view documents.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ Features avaliable:
```
Comment (humdingerb) [inline README.md]: Typo in "avaliable". Maybe just write "Features include:"

**Comment by Nova281** [comment]
Added your suggestions is the latest update on the README.md.

**Comment by humdingerb** [comment]
You have to add the two images as well to your PR. I'd suggest not putting them in an "images" subfolder, but put them directly alongside the readme text file. There's a space in front of a comma in the text.
What do you think about moving the screenshot up before the festures list?

**Pull request closed without merging**